### DISCUSSION TITLE
How to add values of an integer field for multiple entries in a collection?

### DISCUSSION THREAD

**Discussion by tenmillionteeth** [statamic/cms#9205] [Q&A]
We have a client with a collection of entries that include an integer field. They would like to add up the values from that integer field for all entries in the collection that meet filtered criteria, and then display an averaged value. Simple enough, but I'm not sure how to approach it in an Antlers context. 

How would I sum the values from all of the entries?

**Comment by ryanmitchell** [comment]
I would make a custom tag for it, then use the Entry facade to do the query and use ->sum() or ->avg() on the resulting collection.
